site/content/docs/browser-checks/playwright-test.md

```diff
@@ -82,16 +82,15 @@ This is the list of Playwright Test Runner features that are currently supported
 | API testing              | Yes               |
 | Custom fixtures          | Yes               |
 | Reporters                | Only JSON, more to come |
-| Typescript               | Opt-in (Currently in beta - enable it via [Labs](https://app.checklyhq.com/settings/account/labs)) <br> **Not compatible with code sync via GitHub yet.**             |
+| Typescript               | Yes <br> **Not compatible with code sync via GitHub yet.** |
```

Comment:
It's still opt-in via labs for 2022.02 though. Should we mention that?

Reply:
We will release `2023.02` in a matter of weeks. I don't think it's worth mentioning.